GenespotRE/settings.py

```diff
@@ -294,17 +282,34 @@ def GET_BQ_COHORT_SETTINGS():
     'allauth.socialaccount',
     'allauth.socialaccount.providers.google')
 
-TEMPLATE_CONTEXT_PROCESSORS = (
-    'allauth.socialaccount.context_processors.socialaccount',
-    'django.core.context_processors.request',
-    'django.contrib.auth.context_processors.auth',
-    'django.contrib.messages.context_processors.messages',
-    'django.core.context_processors.tz'
-)
-
-TEMPLATE_DIRS += (
-    os.path.join(BASE_DIR, 'templates', 'accounts'),
-    )
+# Template Engine Settings
+TEMPLATES = [
+    {
+        'BACKEND': 'django.template.backends.django.DjangoTemplates',
+        'OPTIONS': {
+            # add any context processors here
+            'context_processors': (
+                'allauth.socialaccount.context_processors.socialaccount',
```

Comment:
I don't remember what I put in originally for context_processors but I found [this](http://django-allauth.readthedocs.org/en/stable/release-notes.html?highlight=context_processor) which makes me think that `allauth.context_processors.allauth` hasn't been around since version 0.6.0 (latest version is 0.25.2, version we currently use is 0.19.1).